R/Surrogate.R

```diff
@@ -27,9 +27,16 @@ Surrogate = R6Class("Surrogate",
     #' @param y_cols (`character()`)\cr
     #' Names of response columns.
     #'
-    #' @return `NULL`
+    #' @return `NULL`.
     update = function(xydt, y_cols) {
-      stop("Abstract")
+      tryCatch(private$.update(xydt, y_cols = y_cols),
```

Comment:
assuming we use an mlr3learner I could also encapuslate there and use a fallback learner? is that more elegant?